standard/expressions.md

```diff
@@ -1480,6 +1487,34 @@ parenthesized_expression
 
 A *parenthesized_expression* is evaluated by evaluating the *expression* within the parentheses. If the *expression* within the parentheses denotes a namespace or type, a compile-time error occurs. Otherwise, the result of the *parenthesized_expression* is the result of the evaluation of the contained *expression*.
 
+### §tuple-expressions-new-clause Tuple expressions
+
+A *tuple_expression* consists of two or more comma-separated and optionally-named *expression*s enclosed in parentheses.
+
+``` ANTLR
+tuple_expression
+    : `(` tuple_element (',' tuple_element)+ ')'
+tuple_element
+    : (identifier ':')? expression
+```
+
+A tuple expression is classified as a tuple. It has a type if all the element expressions have a type. In that case, the type of a tuple expression is the tuple type of the same arity, where each type element has the type - and name, where given - of the corresponding element expression.
+
+A tuple expression is evaluated by evaluating each of its element expressions in order from left to right.
+
+A tuple value can be obtained from a tuple expression by converting it to a tuple type (§implicit-tuple-conversions-new-clause), by reclassifying it as a value ([§11.2.2](expressions.md#1122-values-of-expressions))) or by making it the target of a deconstructing assignment (§deconstructing-assignment-new-clause).
+
+*Example:*
+
+``` c#
+(int i, string) t1 = (i: 1, "One");
+(int i, string) t2 = (i: 2, null);
+var t3 = (i: 3, "Three"); // (int i, string)
+var t4 = (i: 4, null); // Error: no type
+```
+
+In this example, all four tuple expressions are valid. The first two, `t1` and `t2`, do not use the type of the tuple expression, but instead apply an implicit tuple conversion. The third tuple expression has a type `(int i, string)`, and can therefore be reclassified as a value of that type. The declaration of `t4`, on the other hand, is an error: The tuple expression has no type because its second element has no type.
```

Comment:
Might it be useful to change the declaration of `t2` to use `(long x, string)` and note the implicit int to long conversion?

Reply:
Good idea. Fixed in upcoming commit.